src/test/kotlin/no/nav/familie/ks/sak/common/util/TidKtTest.kt

```diff
@@ -0,0 +1,59 @@
+package no.nav.familie.ks.sak.common.util
+
+import org.hamcrest.MatcherAssert.assertThat
+import org.junit.jupiter.api.Test
+import java.time.LocalDate
+import java.time.YearMonth
+import org.hamcrest.CoreMatchers.`is` as Is
+
+internal class TidKtTest {
+
+    @Test
+    fun `LocalDate tilddMMyy() skal formatere dato til ddMMyy format`() {
```

Comment:
testmetoden i familie-tilbake starter med metodenavn som er under test. Skal vi følge det samme her i ks? Da bør alle testene skrives på denne måten

Reply:
Det var tanken her også, men siden det er extension functions (flere klasser som har en tilddMMyy) så prefikset jeg med klasse.

Reply:
Jeg skjønner, det brukte vi i familie-tilbake og gikk veldig fint. Vi bør ha en avtale med andre utviklere også hvis vi har lyst til å bruke dette her, synes jeg

Reply:
Eventuelt så kan man bare splitte opp i en LocalDateTest der man tester extensionfunctions til den klassen, også en separat en for YearMonth